Add table azure_app_service_web_app_slot. closes #567

ParthaI opened the pull request:
# Integration test logs
<details>
  <summary>Logs</summary>
  
```
Add passing integration test logs here
```
</details>

# Example query results
<details>
  <summary>Results</summary>
  
```
+-------------------------------------------------+-------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+---------------------+---------+---------------------------+------------------------+-------------------------------------+-------------+---------+--------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------+----------+----------+---------+-----------------------+------------------+-------------------------+------------------+-----------------------------+---------------------+------------------------------------------------------------------+-------------------------------------------------------------------------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+----------------+----------------+----------------------+-------------------------------------------------------------------+------------+-----------------+----------------------------------------------------------------------------------------------------------+-----------------------------------------------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+----------------------------+-----------------------------+------------------+-------------------------------------------------+-----------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+----------------+--------------------------+-------------------+--------------------------------------+-----------------------------+
| name                                            | app_name                            | id
                            | kind                | state   | type                      | last_modified_time_utc | repository_site_name                | usage_state | enabled | availability_state | server_farm_id
                                                                                                                            | reserved | is_xenon | hyper_v | scm_site_also_stopped | target_swap_slot | client_affinity_enabled | client_cert_mode | client_cert_exclusion_paths | host_names_disabled | custom_domain_verification_id                                    | outbound_ip_addresses                                                                           | possible_outbound_ip_addresses
                                                                                   | container_size | suspended_till | is_default_container | default_host_name                                                 | https_only | redundancy_mode | identity                                                                                                 | host_names                                                            | enabled_host_names
                                                                                      | host_name_ssl_states
                                                                                                    | site_config
                                                                 | traffic_manager_host_names | hosting_environment_profile | slot_swap_status | title                                           | tags
               | akas
                                                                                                                                                         | region         | resource_group           | cloud_environment | subscription_id                      | _ctx                        |
+-------------------------------------------------+-------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+---------------------+---------+---------------------------+------------------------+-------------------------------------+-------------+---------+--------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------+----------+----------+---------+-----------------------+------------------+-------------------------+------------------+-----------------------------+---------------------+------------------------------------------------------------------+-------------------------------------------------------------------------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+----------------+----------------+----------------------+-------------------------------------------------------------------+------------+-----------------+----------------------------------------------------------------------------------------------------------+-----------------------------------------------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+----------------------------+-----------------------------+------------------+-------------------------------------------------+-----------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+----------------+--------------------------+-------------------+--------------------------------------+-----------------------------+
| oauth-app-****-****-asw/new-version | oauth-app-****-****-asw | /subscriptions/****/resourceGroups/app-****-prod-rg/providers/Microsoft.Web/sites/oauth-app-****-****-asw/slots/new-version | app,linux,container | Running | Microsoft.Web/sites/slots | <null>                 | oauth-app-****-****-asw | Normal      | true    | Normal             | /subscriptions/****/resourceGroups/app-****-prod-rg/providers/Microsoft.Web/serverfarms/oauth-app-****-****-asp | true     | false    | false   | false                 | <null>           | true                    | Required         | <null>                      | false               | C2EE6D487****821D****7976 | 20.40.148.89,20.40.148.122,20.40.148.147,20.40.148.153,20.40.148.178,20.40.148.184,20.43.43.32  | 20.40.148.89,20.40.148.122,20.40.148.147,20.40.148.153,20.40.148.178,20.40.148.184,20.40.148.188,20.40.148.201,20.40.148.202,20.40.148.207,20.40.148.216,20.40.148.220,20.40.148.224,20.40.148.227,20.40.148.252,20.40.149.13,20.40.149.18,20.40.149.32,40.66.60.83,40.66.61.60,20.74.16.252,20.74.17.94,20.74.17.20,20.74.17.129,20.43.43.32 | 0              | <null>         | <null>               | oauth-app-****-****-asw-new-version.azurewebsites.net | true       | None
 | {"PrincipalID":"****-0a0f-40a6-9cd4-ca78f8c25e4e","TenantID":"****-3841-412f-abb3-****"} | ["oauth-app-****-****-asw-new-version.azurewebsites.net"] | ["oauth-app-****-****-asw-new-version.azurewebsites.net","oauth-app-****-****-asw-new-version.scm.azurewebsites.net"] | [{"hostType":"Standard","name":"oauth-app-****-****-asw-new-version.azurewebsites.net","sslState":"Disabled"},{"hostType":"Repository","name":"oauth-app-****-****-asw-new-version.scm.azurewebsites.net","sslState":"Disabled"}] | {"alwaysOn":true,"http20Enabled":true,"linuxFxVersion":"DOCKER|****.azurecr.io/authentication/prod/oauth:****.2-0136bfcb","numberOfWorkers":1,"windowsFxVersion":""}          | <null>                     | <null>                      | {}               | oauth-app-****-****-asw/new-version | {"env":"prod","sfx_monitored":"true","stack":"oauth-app"} | ["azure:///subscriptions/****/resourceGroups/app-****-prod-rg/providers/Microsoft.Web/sites/oauth-app-****-****-asw/slots/new-version","azure:///subscriptions/****/resourcegroups/app-****-prod-rg/providers/microsoft.web/sites/oauth-app-****-****-asw/slots/new-version"] | france central | app-****-prod-rg | AzurePublicCloud  | **** | {"connection_name":"azure"} |
| vitrine-****-****-asw/new-version   | vitrine-****-****-asw   | /subscriptions/****/resourceGroups/app-****-prod-rg/providers/Microsoft.Web/sites/vitrine-****-****-asw/slots/new-version   | app,linux           | Running | Microsoft.Web/sites/slots | <null>                 | vitrine-****-****-asw   | Normal      | true    | Normal             | /subscriptions/****/resourceGroups/app-****-prod-rg/providers/Microsoft.Web/serverfarms/vitrine-****-****-asp   | true     | false    | false   | false                 | <null>           | true                    | Required         | <null>                      | false               | C2EE6D487****821D****7976 | 20.40.148.188,20.40.148.201,20.40.148.202,20.40.148.207,20.40.148.216,20.40.148.220,20.43.43.32 | 20.40.148.89,20.40.148.122,20.40.148.147,20.40.148.153,20.40.148.178,20.40.148.184,20.40.148.188,20.40.148.201,20.40.148.202,20.40.148.207,20.40.148.216,20.40.148.220,20.40.148.224,20.40.148.227,20.40.148.252,20.40.149.13,20.40.149.18,20.40.149.32,40.66.60.83,40.66.61.60,20.74.16.252,20.74.17.94,20.74.17.20,20.74.17.129,20.43.43.32 | 0              | <null>         | <null>               | vitrine-****-****-asw-new-version.azurewebsites.net   | true       | None
 | {"PrincipalID":"****-12d1-4b40-8b4b-3e2c07c45649","TenantID":"****-3841-412f-abb3-****"} | ["vitrine-****-****-asw-new-version.azurewebsites.net"]   | ["vitrine-****-****-asw-new-version.azurewebsites.net","vitrine-****-****-asw-new-version.scm.azurewebsites.net"]     | [{"hostType":"Standard","name":"vitrine-****-****-asw-new-version.azurewebsites.net","sslState":"Disabled"},{"hostType":"Repository","name":"vitrine-****-****-asw-new-version.scm.azurewebsites.net","sslState":"Disabled"}]     | {"alwaysOn":true,"http20Enabled":true,"linuxFxVersion":"PHP|8.2","numberOfWorkers":1,"windowsFxVersion":""}
                                                                 | <null>                     | <null>                      | {}               | vitrine-****-****-asw/new-version   | {"env":"prod","sfx_monitored":"true","stack":"vitrine"}   | ["azure:///subscriptions/****/resourceGroups/app-****-prod-rg/providers/Microsoft.Web/sites/vitrine-****-****-asw/slots/new-version","azure:///subscriptions/****/resourcegroups/app-****-prod-rg/providers/microsoft.web/sites/vitrine-****-****-asw/slots/new-version"]     | france central | app-****-prod-rg | AzurePublicCloud  | **** | {"connection_name":"azure"} |
| api-app-****-****-asw/new-version   | api-app-****-****-asw   | /subscriptions/****/resourceGroups/app-****-prod-rg/providers/Microsoft.Web/sites/api-app-****-****-asw/slots/new-version   | app,linux,container | Running | Microsoft.Web/sites/slots | <null>                 | api-app-****-****-asw   | Normal      | true    | Normal             | /subscriptions/****/resourceGroups/app-****-prod-rg/providers/Microsoft.Web/serverfarms/api-app-****-****-asp   | true     | false    | false   | false                 | <null>           | true                    | Required         | <null>                      | false               | C2EE6D487****821D****7976 | 20.40.148.89,20.40.148.122,20.40.148.147,20.40.148.153,20.40.148.178,20.40.148.184,20.43.43.32  | 20.40.148.89,20.40.148.122,20.40.148.147,20.40.148.153,20.40.148.178,20.40.148.184,20.40.148.188,20.40.148.201,20.40.148.202,20.40.148.207,20.40.148.216,20.40.148.220,20.40.148.224,20.40.148.227,20.40.148.252,20.40.149.13,20.40.149.18,20.40.149.32,40.66.60.83,40.66.61.60,20.74.16.252,20.74.17.94,20.74.17.20,20.74.17.129,20.43.43.32 | 0              | <null>         | <null>               | api-app-****-****-asw-new-version.azurewebsites.net   | true       | None
 | {"PrincipalID":"****-2409-4b20-b758-****","TenantID":"****-3841-412f-abb3-****"} | ["api-app-****-****-asw-new-version.azurewebsites.net"]   | ["api-app-****-****-asw-new-version.azurewebsites.net","api-app-****-****-asw-new-version.scm.azurewebsites.net"]     | [{"hostType":"Standard","name":"api-app-****-****-asw-new-version.azurewebsites.net","sslState":"Disabled"},{"hostType":"Repository","name":"api-app-****-****-asw-new-version.scm.azurewebsites.net","sslState":"Disabled"}]     | {"alwaysOn":true,"http20Enabled":true,"linuxFxVersion":"DOCKER|****.azurecr.io/api-****/prod/api-platform:****.1-****","numberOfWorkers":1,"windowsFxVersion":""} | <null>                     | <null>                      | {}               | api-app-****-****-asw/new-version   | {"env":"prod","sfx_monitored":"true","stack":"api-app"}   | ["azure:///subscriptions/****/resourceGroups/app-****-prod-rg/providers/Microsoft.Web/sites/api-app-****-****-asw/slots/new-version","azure:///subscriptions/****/resourcegroups/app-****-prod-rg/providers/microsoft.web/sites/api-app-****-****-asw/slots/new-version"]     | france central | app-****-prod-rg | AzurePublicCloud  | **** | {"connection_name":"azure"} |
+-------------------------------------------------+-------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+---------------------+---------+---------------------------+------------------------+-------------------------------------+-------------+---------+--------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------+----------+----------+---------+-----------------------+------------------+-------------------------+------------------+-----------------------------+---------------------+------------------------------------------------------------------+-------------------------------------------------------------------------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+----------------+----------------+----------------------+-------------------------------------------------------------------+------------+-----------------+----------------------------------------------------------------------------------------------------------+-----------------------------------------------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+----------------------------+-----------------------------+------------------+-------------------------------------------------+-----------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+----------------+--------------------------+-------------------+--------------------------------------+-----------------------------+
```
</details>

Shr3ps commented:
> Hi @Shr3ps based on your [issue comment](https://github.com/turbot/steampipe-plugin-azure/issues/567#issue-1581955769), we have added a new table `azure_app_service_web_app_slot ` in this PR. It would be really appreciated if you could give it a try and share your experience with us. Thanks!

Hello, thanks for this PR!
I just tried on a Subscription and got this error:

```
Welcome to Steampipe v0.19.3
For more information, type .help
> select * from azure_app_service_web_app_slot

Error: interfaceToColumnValue failed for column 'last_modified_time_utc': cannot convert 2023-03-27T12:08:49.8633333Z to a time.Time (SQLSTATE HV000)
```

ParthaI commented:
> > Hi @Shr3ps based on your [issue comment](https://github.com/turbot/steampipe-plugin-azure/issues/567#issue-1581955769), we have added a new table `azure_app_service_web_app_slot ` in this PR. It would be really appreciated if you could give it a try and share your experience with us. Thanks!
> 
> Hello, thanks for this PR! I just tried on a Subscription and got this error:
> 
> ```
> Welcome to Steampipe v0.19.3
> For more information, type .help
> > select * from azure_app_service_web_app_slot
> 
> Error: interfaceToColumnValue failed for column 'last_modified_time_utc': cannot convert 2023-03-27T12:08:49.8633333Z to a time.Time (SQLSTATE HV000)
> ```

@Shr3ps Apologize for putting you in trouble, It was my bad, I have updated the column `last_modified_time_utc`.
Please pull the latest changes to your local then run the `make` command then run the query.
If you are running in to any other issues please let me know.

Thanks!

ParthaI commented:
> 

@Shr3ps Thanks for the confirmation. Could you please share the query output here?

Shr3ps commented:
+-------------------------------------------------+-------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+---------------------+---------+---------------------------+------------------------+-------------------------------------+-------------+---------+--------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------+----------+----------+---------+-----------------------+------------------+-------------------------+------------------+-----------------------------+---------------------+------------------------------------------------------------------+-------------------------------------------------------------------------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+----------------+----------------+----------------------+-------------------------------------------------------------------+------------+-----------------+----------------------------------------------------------------------------------------------------------+-----------------------------------------------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+----------------------------+-----------------------------+------------------+-------------------------------------------------+-----------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+----------------+--------------------------+-------------------+--------------------------------------+-----------------------------+
| name                                            | app_name                            | id
                            | kind                | state   | type                      | last_modified_time_utc | repository_site_name                | usage_state | enabled | availability_state | server_farm_id
                                                                                                                            | reserved | is_xenon | hyper_v | scm_site_also_stopped | target_swap_slot | client_affinity_enabled | client_cert_mode | client_cert_exclusion_paths | host_names_disabled | custom_domain_verification_id                                    | outbound_ip_addresses                                                                           | possible_outbound_ip_addresses
                                                                                   | container_size | suspended_till | is_default_container | default_host_name                                                 | https_only | redundancy_mode | identity                                                                                                 | host_names                                                            | enabled_host_names
                                                                                      | host_name_ssl_states
                                                                                                    | site_config
                                                                 | traffic_manager_host_names | hosting_environment_profile | slot_swap_status | title                                           | tags
               | akas
                                                                                                                                                         | region         | resource_group           | cloud_environment | subscription_id                      | _ctx                        |
+-------------------------------------------------+-------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+---------------------+---------+---------------------------+------------------------+-------------------------------------+-------------+---------+--------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------+----------+----------+---------+-----------------------+------------------+-------------------------+------------------+-----------------------------+---------------------+------------------------------------------------------------------+-------------------------------------------------------------------------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+----------------+----------------+----------------------+-------------------------------------------------------------------+------------+-----------------+----------------------------------------------------------------------------------------------------------+-----------------------------------------------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+----------------------------+-----------------------------+------------------+-------------------------------------------------+-----------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+----------------+--------------------------+-------------------+--------------------------------------+-----------------------------+
| oauth-app-****-****-asw/new-version | oauth-app-****-****-asw | /subscriptions/****/resourceGroups/app-****-prod-rg/providers/Microsoft.Web/sites/oauth-app-****-****-asw/slots/new-version | app,linux,container | Running | Microsoft.Web/sites/slots | <null>                 | oauth-app-****-****-asw | Normal      | true    | Normal             | /subscriptions/****/resourceGroups/app-****-prod-rg/providers/Microsoft.Web/serverfarms/oauth-app-****-****-asp | true     | false    | false   | false                 | <null>           | true                    | Required         | <null>                      | false               | C2EE6D487****821D****7976 | 20.40.148.89,20.40.148.122,20.40.148.147,20.40.148.153,20.40.148.178,20.40.148.184,20.43.43.32  | 20.40.148.89,20.40.148.122,20.40.148.147,20.40.148.153,20.40.148.178,20.40.148.184,20.40.148.188,20.40.148.201,20.40.148.202,20.40.148.207,20.40.148.216,20.40.148.220,20.40.148.224,20.40.148.227,20.40.148.252,20.40.149.13,20.40.149.18,20.40.149.32,40.66.60.83,40.66.61.60,20.74.16.252,20.74.17.94,20.74.17.20,20.74.17.129,20.43.43.32 | 0              | <null>         | <null>               | oauth-app-****-****-asw-new-version.azurewebsites.net | true       | None
 | {"PrincipalID":"****-0a0f-40a6-9cd4-ca78f8c25e4e","TenantID":"****-3841-412f-abb3-****"} | ["oauth-app-****-****-asw-new-version.azurewebsites.net"] | ["oauth-app-****-****-asw-new-version.azurewebsites.net","oauth-app-****-****-asw-new-version.scm.azurewebsites.net"] | [{"hostType":"Standard","name":"oauth-app-****-****-asw-new-version.azurewebsites.net","sslState":"Disabled"},{"hostType":"Repository","name":"oauth-app-****-****-asw-new-version.scm.azurewebsites.net","sslState":"Disabled"}] | {"alwaysOn":true,"http20Enabled":true,"linuxFxVersion":"DOCKER|****.azurecr.io/authentication/prod/oauth:****.2-0136bfcb","numberOfWorkers":1,"windowsFxVersion":""}          | <null>                     | <null>                      | {}               | oauth-app-****-****-asw/new-version | {"env":"prod","sfx_monitored":"true","stack":"oauth-app"} | ["azure:///subscriptions/****/resourceGroups/app-****-prod-rg/providers/Microsoft.Web/sites/oauth-app-****-****-asw/slots/new-version","azure:///subscriptions/****/resourcegroups/app-****-prod-rg/providers/microsoft.web/sites/oauth-app-****-****-asw/slots/new-version"] | france central | app-****-prod-rg | AzurePublicCloud  | **** | {"connection_name":"azure"} |
| vitrine-****-****-asw/new-version   | vitrine-****-****-asw   | /subscriptions/****/resourceGroups/app-****-prod-rg/providers/Microsoft.Web/sites/vitrine-****-****-asw/slots/new-version   | app,linux           | Running | Microsoft.Web/sites/slots | <null>                 | vitrine-****-****-asw   | Normal      | true    | Normal             | /subscriptions/****/resourceGroups/app-****-prod-rg/providers/Microsoft.Web/serverfarms/vitrine-****-****-asp   | true     | false    | false   | false                 | <null>           | true                    | Required         | <null>                      | false               | C2EE6D487****821D****7976 | 20.40.148.188,20.40.148.201,20.40.148.202,20.40.148.207,20.40.148.216,20.40.148.220,20.43.43.32 | 20.40.148.89,20.40.148.122,20.40.148.147,20.40.148.153,20.40.148.178,20.40.148.184,20.40.148.188,20.40.148.201,20.40.148.202,20.40.148.207,20.40.148.216,20.40.148.220,20.40.148.224,20.40.148.227,20.40.148.252,20.40.149.13,20.40.149.18,20.40.149.32,40.66.60.83,40.66.61.60,20.74.16.252,20.74.17.94,20.74.17.20,20.74.17.129,20.43.43.32 | 0              | <null>         | <null>               | vitrine-****-****-asw-new-version.azurewebsites.net   | true       | None
 | {"PrincipalID":"****-12d1-4b40-8b4b-3e2c07c45649","TenantID":"****-3841-412f-abb3-****"} | ["vitrine-****-****-asw-new-version.azurewebsites.net"]   | ["vitrine-****-****-asw-new-version.azurewebsites.net","vitrine-****-****-asw-new-version.scm.azurewebsites.net"]     | [{"hostType":"Standard","name":"vitrine-****-****-asw-new-version.azurewebsites.net","sslState":"Disabled"},{"hostType":"Repository","name":"vitrine-****-****-asw-new-version.scm.azurewebsites.net","sslState":"Disabled"}]     | {"alwaysOn":true,"http20Enabled":true,"linuxFxVersion":"PHP|8.2","numberOfWorkers":1,"windowsFxVersion":""}
                                                                 | <null>                     | <null>                      | {}               | vitrine-****-****-asw/new-version   | {"env":"prod","sfx_monitored":"true","stack":"vitrine"}   | ["azure:///subscriptions/****/resourceGroups/app-****-prod-rg/providers/Microsoft.Web/sites/vitrine-****-****-asw/slots/new-version","azure:///subscriptions/****/resourcegroups/app-****-prod-rg/providers/microsoft.web/sites/vitrine-****-****-asw/slots/new-version"]     | france central | app-****-prod-rg | AzurePublicCloud  | **** | {"connection_name":"azure"} |
| api-app-****-****-asw/new-version   | api-app-****-****-asw   | /subscriptions/****/resourceGroups/app-****-prod-rg/providers/Microsoft.Web/sites/api-app-****-****-asw/slots/new-version   | app,linux,container | Running | Microsoft.Web/sites/slots | <null>                 | api-app-****-****-asw   | Normal      | true    | Normal             | /subscriptions/****/resourceGroups/app-****-prod-rg/providers/Microsoft.Web/serverfarms/api-app-****-****-asp   | true     | false    | false   | false                 | <null>           | true                    | Required         | <null>                      | false               | C2EE6D487****821D****7976 | 20.40.148.89,20.40.148.122,20.40.148.147,20.40.148.153,20.40.148.178,20.40.148.184,20.43.43.32  | 20.40.148.89,20.40.148.122,20.40.148.147,20.40.148.153,20.40.148.178,20.40.148.184,20.40.148.188,20.40.148.201,20.40.148.202,20.40.148.207,20.40.148.216,20.40.148.220,20.40.148.224,20.40.148.227,20.40.148.252,20.40.149.13,20.40.149.18,20.40.149.32,40.66.60.83,40.66.61.60,20.74.16.252,20.74.17.94,20.74.17.20,20.74.17.129,20.43.43.32 | 0              | <null>         | <null>               | api-app-****-****-asw-new-version.azurewebsites.net   | true       | None
 | {"PrincipalID":"****-2409-4b20-b758-****","TenantID":"****-3841-412f-abb3-****"} | ["api-app-****-****-asw-new-version.azurewebsites.net"]   | ["api-app-****-****-asw-new-version.azurewebsites.net","api-app-****-****-asw-new-version.scm.azurewebsites.net"]     | [{"hostType":"Standard","name":"api-app-****-****-asw-new-version.azurewebsites.net","sslState":"Disabled"},{"hostType":"Repository","name":"api-app-****-****-asw-new-version.scm.azurewebsites.net","sslState":"Disabled"}]     | {"alwaysOn":true,"http20Enabled":true,"linuxFxVersion":"DOCKER|****.azurecr.io/api-****/prod/api-platform:****.1-****","numberOfWorkers":1,"windowsFxVersion":""} | <null>                     | <null>                      | {}               | api-app-****-****-asw/new-version   | {"env":"prod","sfx_monitored":"true","stack":"api-app"}   | ["azure:///subscriptions/****/resourceGroups/app-****-prod-rg/providers/Microsoft.Web/sites/api-app-****-****-asw/slots/new-version","azure:///subscriptions/****/resourcegroups/app-****-prod-rg/providers/microsoft.web/sites/api-app-****-****-asw/slots/new-version"]     | france central | app-****-prod-rg | AzurePublicCloud  | **** | {"connection_name":"azure"} |
+-------------------------------------------------+-------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+---------------------+---------+---------------------------+------------------------+-------------------------------------+-------------+---------+--------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------+----------+----------+---------+-----------------------+------------------+-------------------------+------------------+-----------------------------+---------------------+------------------------------------------------------------------+-------------------------------------------------------------------------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+----------------+----------------+----------------------+-------------------------------------------------------------------+------------+-----------------+----------------------------------------------------------------------------------------------------------+-----------------------------------------------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+----------------------------+-----------------------------+------------------+-------------------------------------------------+-----------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+----------------+--------------------------+-------------------+--------------------------------------+-----------------------------+

misraved commented on docs/tables/azure_app_service_web_app_slot.md:
```diff
@@ -0,0 +1,124 @@
+# Table: azure_app_service_web_app
+
+When you deploy your web app, web app on Linux, mobile back end, or API app to Azure App Service, you can use a separate deployment slot instead of the default production slot when you're running in the Standard, Premium, or Isolated App Service plan tier. Deployment slots are live apps with their own host names. App content and configurations elements can be swapped between two deployment slots, ...
```
```suggestion
When you deploy your web app on Linux, mobile back end, or API app to Azure App Service, you can use a separate deployment slot instead of the default production slot when running in the Standard, Premium, or Isolated App Service plan tier. Deployment slots are live apps with their host names. App content and configuration elements can be swapped between two deployment slots, including the production slot.
```

misraved commented on docs/tables/azure_app_service_web_app_slot.md:
````diff
@@ -0,0 +1,124 @@
+  enabled;
+```
+
+### List slots which accepts HTTP traffics (i.e HTTPS only is disabled)
````
```suggestion
### List slots that accepts HTTP traffic (i.e only HTTPS is disabled)
```

misraved commented on docs/tables/azure_app_service_web_app_slot.md:
````diff
@@ -0,0 +1,124 @@
+  azure_app_service_web_app_slot;
+```
+
+### List slots that are enabled
````
```suggestion
### List slots where the apps are enabled
```

misraved reviewed (commented):
@ParthaI please take a look at the review comments. Thanks!!

ParthaI commented on azure/table_azure_app_service_web_app_slot.go:
```diff
@@ -0,0 +1,452 @@
+			{
+				Name:        "outbound_ip_addresses",
+				Description: "List of IP addresses that the app uses for outbound connections (e.g. database access). Includes VIPs from tenants that site can be hosted with current settings.",
+				Type:        proto.ColumnType_STRING,
```
In doc it is mentioned that the data type of this property is string.
https://learn.microsoft.com/en-us/rest/api/appservice/web-apps/list-slots#site

ParthaI commented on azure/table_azure_app_service_web_app_slot.go:
```diff
@@ -0,0 +1,452 @@
+			{
+				Name:        "possible_outbound_ip_addresses",
+				Description: "List of IP addresses that the app uses for outbound connections (e.g. database access). Includes VIPs from all tenants except dataComponent.",
+				Type:        proto.ColumnType_STRING,
```
In doc it is mentioned that the data type of this property is string.
https://learn.microsoft.com/en-us/rest/api/appservice/web-apps/list-slots#site

ParthaI commented on azure/table_azure_app_service_web_app_slot.go:
```diff
@@ -0,0 +1,452 @@
+			},
+			{
+				Name:        "is_xenon",
+				Description: "Hyper-V sandbox.",
```
In [doc](https://learn.microsoft.com/en-us/rest/api/appservice/web-apps/list-slots#site) the description for the column is `Obsolete: Hyper-V sandbox.`
